pages/pipelines/hosted_agents/linux.md

```diff
@@ -28,26 +28,29 @@ To configure your Linux instance you can use the [Docker Compose](https://github
 
 ## Cache volumes
 
-Cache Volumes are external volumes attached to hosted agent instances. Cache Volumes are attached on a best-effort basis depending on their locality, expiration and current usage (so they should not be relied upon as durable data storage).
+_Cache volumes_ are external volumes attached to hosted agent instances. These volumes are attached on a best-effort basis depending on their locality, expiration and current usage, and therefore, should not be relied upon as durable data storage.
 
-Cache Volumes are disabled by default and can be enabled by providing a list of paths to cache at the pipeline level or the step level. Cache Volumes are scoped to a pipeline and are shared between all steps in a pipeline by default.
+By default, cache volumes:
 
-Cache Volumes act as regular disks with the following properties:
+- Are disabled, although you can enable them by providing a list of paths to cache at the pipeline- or step-level.
+- Are scoped to a pipeline and are shared between all steps in the pipeline.
 
-- They're backed by local NVMe storage. You can expect high performance.
-- A Cache Volume is formatted as a regular Linux filesystem (e.g. ext4), so you can expect them to support any use-case you have that Linux supports.
+Cache volumes act as regular disks with the following properties:
+
+- The volumes backed by local NVMe storage, delivering high performance.
```

Comment:
What exactly is meant by "backed by" and where exactly is the "local" NVMe storage kept?

Does "backed by" mean use?

Reply:
Means 'uses' in this context.